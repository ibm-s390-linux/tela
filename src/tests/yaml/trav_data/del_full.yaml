delete:
delete:
delete:
delete:
  delete:
    delete:
delete:
delete:
delete:
