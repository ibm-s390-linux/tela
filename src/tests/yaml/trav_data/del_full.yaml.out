Before (root=non-null):
==================================
  delete:
  delete:
  delete:
  delete:
    delete:
      delete:
  delete:
  delete:
  delete:
==================================

Callback:
==================================
delete                                  : delete:
*** Deleting node
delete                                  : delete:
*** Deleting node
delete                                  : delete:
*** Deleting node
delete                                  : delete:
*** Deleting node
delete                                  : delete:
*** Deleting node
delete                                  : delete:
*** Deleting node
delete                                  : delete:
*** Deleting node
==================================

After (root=null):
==================================
==================================
